Detectors/Vertexing/src/SVertexer.cxx

```diff
@@ -12,41 +12,54 @@
 /// \file SVertexer.cxx
 /// \brief Secondary vertex finder
 /// \author ruben.shahoyan@cern.ch
-
+#include <iostream>
+#include <fstream>
 #include "DetectorsVertexing/SVertexer.h"
 #include "DetectorsBase/Propagator.h"
 #include "TPCReconstruction/TPCFastTransformHelperO2.h"
 #include "DataFormatsTPC/VDriftCorrFact.h"
 #include "CorrectionMapsHelper.h"
-#include "Framework/ProcessingContext.h"
-#include "Framework/DataProcessorSpec.h"
-#include "ReconstructionDataFormats/StrangeTrack.h"
 
 #ifdef WITH_OPENMP
 #include <omp.h>
 #endif
 
 #include "ReconstructionDataFormats/GlobalTrackID.h"
-
 using namespace o2::vertexing;
-namespace o2f = o2::framework;
+
 using PID = o2::track::PID;
 using TrackTPCITS = o2::dataformats::TrackTPCITS;
 using TrackITS = o2::its::TrackITS;
 using TrackTPC = o2::tpc::TrackTPC;
+using VBracket = o2::math_utils::Bracket<int>;
+
+std::ofstream log_file(
+  "log_file.txt", std::ios_base::out | std::ios_base::app);
+std::ofstream log_pT4Hlambdalambda(
+  "log_pT4Hlambdalambda.txt", std::ios_base::out | std::ios_base::app);
+std::ofstream log_pT4He3Lambda(
+  "log_pT4He3Lambda.txt", std::ios_base::out | std::ios_base::app);
+std::ofstream log_pTpion(
+  "log_pTpion.txt", std::ios_base::out | std::ios_base::app);
+
+std::ofstream log_pT3body(
+  "log_pT3body.txt", std::ios_base::out | std::ios_base::app);
```

Comment:
This is not used anywhere and we should not produce a stray log files in the standard mode